PKGBUILD

```diff
@@ -46,11 +47,19 @@ ros_depends=(ros-noetic-tf2-ros
   ros-noetic-tf2
   ros-noetic-eigen-conversions)
 depends=(${ros_depends[@]}
+  geographiclib
   eigen)
 
 _dir="robot_localization-${pkgver}"
-source=("${pkgname}-${pkgver}.tar.gz"::"https://github.com/cra-ros-pkg/robot_localization/archive/${pkgver}.tar.gz")
-sha256sums=('9ca03e611b3c0d024dca3642a8eb0da75a8a776c6dee3082717a314e41fbafa8')
+source=("${pkgname}-${pkgver}.tar.gz"::"https://github.com/cra-ros-pkg/robot_localization/archive/${pkgver}.tar.gz"
+        "624.patch"::"https://github.com/cra-ros-pkg/robot_localization/pull/624.patch")
```

Comment:
Can you make a corresponding issue on this repo for this error? It doesn't have to be detailed and can just be a link to an associated upstream issue. We would just like to have it for tracking purposes.

Reply:
The issue is #2

Reply:
thx!